src/main/java/de/tum/cit/aet/artemis/core/web/LectureResource.java

```diff
@@ -255,8 +255,7 @@ public ResponseEntity<Lecture> importLecture(@PathVariable long sourceLectureId,
         authCheckService.checkHasAtLeastRoleForLectureElseThrow(Role.EDITOR, sourceLecture, user);
         authCheckService.checkHasAtLeastRoleInCourseElseThrow(Role.EDITOR, destinationCourse, user);
 
-        final var savedLecture = lectureImportService.importLecture(sourceLecture, destinationCourse);
-        channelService.createLectureChannel(savedLecture, Optional.empty());
```

Comment:
This is now handled in `lectureImportService.importLecture` to avoid duplication when importing the lectures in the new `LearningObjectImportService`